kubernetes/kong-gw/kong.yml

```diff
@@ -55,23 +53,26 @@ spec:
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: integration-service-others
+  name: integration-service-info
+  namespace: kong
+  annotations:
+    konghq.com/strip-path: "true"
 spec:
```

Comment:
This ingress seems not working (don't know why).
Please check the configuration of the second Ingress definition.